frontend/interfaces/software.ts

```diff
@@ -44,6 +44,7 @@ export interface ISoftware {
   installed_paths?: string[];
   browser?: string;
   vendor?: string;
+  icon_url?: string; // Only available on team view if an admin uploaded an icon to a team's software
```

Comment:
here the type is `string | undefined` but on ISoftwareTitle and IHostSoftware the type is `string | null`? Just making sure the typing is correct here